runtime/src/lib.rs

```diff
@@ -87,7 +87,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
     // implementation changes and behavior does not, then leave spec_version as
     // is and increment impl_version.
     spec_version: 228,
-    impl_version: 5,
+    impl_version: 6,
```

Comment:
I'd argue that this is not just an implementation change (such as a refactor), but rather a change in runtime behavior. Can we implement spec_version instead?

Reply:
We havent released this yet, we will package it with this version.

Reply:
👍